producer-c/producer-cloudwatch-integ/jobs/orchestrator.groovy

```diff
@@ -0,0 +1,160 @@
+import jenkins.model.*
+
+WORKSPACE_PRODUCER="producer-c/producer-cloudwatch-integ"
+WORKSPACE_CONSUMER="consumer-java/aws-kinesis-video-producer-sdk-canary-consumer"
+GIT_URL='https://github.com/aws-samples/amazon-kinesis-video-streams-demos.git'
+GIT_HASH='producer'
```

Comment:
this should be master

Reply:
It will change once the PR is approved.

Reply:
Will change it once PR is approved. I am running Jenkins with this branch right now.